LICENSE.txt

```diff
@@ -88,9 +88,11 @@ Copyright (C) 2022 "RetroGamerFan"
 Copyright (C) 2022 "tomracer"
 Copyright (C) 2022 "TotalCaesar659"
 Copyright (C) 2022 Manuel K
-Copyright (C) 2022 lokpro
+Copyright (C) 2022-2023 lokpro
 Copyirght (C) 2022 Ryan "ryan-edgewurth" Penfold
 Copyright (C) 2022 "SilPho"
+Copyright (C) 2023 "dfghjkjhgr" NegativeSector
```

Comment:
Pretty sure NegativeSector isn't a real name either...

Reply:
@dfghjkjhgr do you have any preference for how you're identified in the license file?

Reply:
```suggestion
Copyright (C) 2023 "dfghjkjhgr"
```